NorthstarDedicatedTest/scriptUtility.cpp

```diff
@@ -0,0 +1,28 @@
+#include "pch.h"
```

Comment:
capitalization of filename should be more inline with other files

Reply:
ok but we should also make a folder for all script additions. They slowly get out of hand